generators/bdk/index.js

```diff
@@ -19,6 +20,15 @@ module.exports = class extends Generator {
     this.log('Welcome to Symphony Generator '.gray + `v${packageJson.version}`.yellow);
     this.log('Application files will be generated in folder: '.gray + `${this.destinationRoot()}`.yellow);
     this.log('______________________________________________________________________________________________________'.yellow);
+
+    try {
+      const arrayOfFiles = fs.readdirSync(this.destinationRoot());
+      if (arrayOfFiles.length > 0) {
+        this.log(`(!) Folder ${this.destinationRoot()} is not empty. Are you sure you want to continue?`.red);
```

Comment:
Here we're just warning that folder is empty. We could also let user change the folder and check again. see comment in ticket https://perzoinc.atlassian.net/browse/PLAT-10173